models/op/governance/optimism/op_governance_optimism_delegators_incremental.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+        tags=['dunesql']
+        , schema = 'op_governance_optimism'
+        , alias = alias('delegators_incremental')
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['block_time', 'tx_hash', 'evt_index']
+  )
+}}
+
+
+SELECT evt_tx_hash AS tx_hash, 
+evt_block_time AS block_time, 
+evt_block_number AS block_number,
+evt_index,
+fromDelegate,
+toDelegate
+FROM  {{ source('op_optimism', 'GovernanceToken_evt_DelegateChanged') }}
+{% if is_incremental() %}
+    WHERE evt_block_time >= DATE_TRUNC('day', NOW() - INTERVAL '7' DAY)
```

Comment:
By the way we can use `incremental_predicate` to rewrite all incremental filters:
i.e.
```suggestion
    WHERE {{ incremental_predicate('evt_block_time') }}
```